spec/wiremock/prison-api/mappings/elite2api_api_bookings_offenderno_alerts-88bf8e3c-ca15-4af2-98f6-022dcb9f72f2.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id" : "88bf8e3c-ca15-4af2-98f6-022dcb9f72f2",
+  "name" : "elite2api_api_bookings_offenderno_alerts",
```

Comment:
A  minor comment (and I suspect this is also down to the recording). Filenames refer to elite2 which is in the midst of change in the near future.  Perhaps something to add to a todo/list backlog (or just ignore me of course).

Reply:
Hi Matt, the filenames are coming automatically from the URLs - I'll regenerate these recordings when we switch over to the new domain (September?)